aiida/orm/utils/mixins.py

```diff
@@ -204,67 +194,3 @@ def delete_attribute(self, key):
-        :raises TypeError: if the path is not a string and relative path.
-        :raises aiida.common.exceptions.ModificationNotAllowed: when the node is sealed and therefore immutable.
-        """
-        self.check_mutability()
-
-        if isinstance(handle, io.StringIO):
-            handle = io.BytesIO(handle.read().encode('utf-8'))
-
-        if isinstance(handle, tempfile._TemporaryFileWrapper):  # pylint: disable=protected-access
-            if 'b' in handle.file.mode:
-                handle = io.BytesIO(handle.read())
-            else:
-                handle = io.BytesIO(handle.read().encode('utf-8'))
-
-        self._repository.put_object_from_filelike(handle, path)
-        self._update_repository_metadata()
-
-    @override
-    def put_object_from_file(self, filepath: str, path: str):
-        """Store a new object under `path` with contents of the file located at `filepath` on the local file system.
-
-        :param filepath: absolute path of file whose contents to copy to the repository
-        :param path: the relative path where to store the object in the repository.
-        :raises TypeError: if the path is not a string and relative path, or the handle is not a byte stream.
-        :raises aiida.common.exceptions.ModificationNotAllowed: when the node is sealed and therefore immutable.
-        """
-        self.check_mutability()
-        self._repository.put_object_from_file(filepath, path)
-        self._update_repository_metadata()
-
-    @override
-    def put_object_from_tree(self, filepath: str, path: str = None):
-        """Store the entire contents of `filepath` on the local file system in the repository with under given `path`.
-
-        :param filepath: absolute path of the directory whose contents to copy to the repository.
-        :param path: the relative path where to store the objects in the repository.
-        :raises TypeError: if the path is not a string and relative path.
-        :raises aiida.common.exceptions.ModificationNotAllowed: when the node is sealed and therefore immutable.
-        """
-        self.check_mutability()
-        self._repository.put_object_from_tree(filepath, path)
-        self._update_repository_metadata()
-
-    @override
-    def delete_object(self, path: str):
-        """Delete the object from the repository.
-
-        :param key: fully qualified identifier for the object within the repository.
-        :raises TypeError: if the path is not a string and relative path.
-        :raises FileNotFoundError: if the file does not exist.
-        :raises IsADirectoryError: if the object is a directory and not a file.
-        :raises OSError: if the file could not be deleted.
-        :raises aiida.common.exceptions.ModificationNotAllowed: when the node is sealed and therefore immutable.
-        """
-        self.check_mutability()
-        self._repository.delete_object(path)
-        self._update_repository_metadata()
```

Comment:
Unless I am missing something, none of these methods, actually changed anything from the ones they are overriding!?
Perhaps the `check_mutability` was meant to also check for whether the node is sealed, but that is certainly not the case at present?